### DISCUSSION TITLE
I'm starting to build APIs and need some resources that are more detailed or generator for JAVA

### DISCUSSION THREAD

**Discussion by alexmaeghan** [pactumjs/pactum#132] [General]
Hello! I am starting to work on JAVA script, which I've never worked with before. I've tried looking at examples/resources but I'm simply confused by some of the contradictory code. I need help with credible resources for the following APIs. 
- Inventory API
- Order API
- Product API
- Profile API
- Transactions API
- WEBhook Subscription API

**Comment by ASaiAnudeep** [comment]
Hello @alexmaeghan 

- Are you trying to build API's or wanted to test API's? 
- Can you give us more details on contradictory code.
